localstack/services/cloudwatch/provider.py

```diff
@@ -103,29 +104,29 @@ def update_state(target, self, reason, reason_data, state_value):
 def put_metric_alarm(
     target,
     self,
-    name,
-    namespace,
-    metric_name,
-    metric_data_queries,
-    comparison_operator,
-    evaluation_periods,
-    datapoints_to_alarm,
-    period,
-    threshold,
-    statistic,
-    extended_statistic,
-    description,
-    dimensions,
-    alarm_actions,
-    ok_actions,
-    insufficient_data_actions,
-    unit,
-    actions_enabled,
-    treat_missing_data,
-    evaluate_low_sample_count_percentile,
-    threshold_metric_id,
-    rule=None,
-    tags=None,
```

Comment:
Signature updated to reflect the changes in https://github.com/getmoto/moto/pull/7560